transcom/mymove
# B 18902 i 12681 updating destination address

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A907944)
## [Issue ticket](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A927471)

Need to allow shipment destination address changes when destination SIT service items are requested but not yet approved. Before they're approved, we treat the shipment as a direct delivery shipment. Since it is direct delivery, we won't show the TOO any sit distances at the point of reviewing the destination address change. Rather, they can view that when approving the destination SIT service items.

To test:
I used BILWEI as the move.

1. Request destination SIT but do NOT approve them as the TOO.
2. As prime, change the shipment address to a zip that will require a TOO review (more than 50 miles from original zip)
3. You should now be able to update the destination address as prime. 
4. Review the address change as TOO.
5. Verify address was updated on prime sim.
6. Approve destination SIT service items as TOO.
7. Make another address change > 50 miles.
8. Review and approve as TOO. You will see the sit calculations now, since it is no longer treated as a direct delivery after we approved the destination service items.
